app/controllers/reports_controller.rb

```diff
@@ -7,7 +7,7 @@ class ReportsController < ApplicationController
   def authorize_read_report() authorize! :read, :report end
 
   def atn_reg_sheets
-    @attendee_attr_names = %w[aga_id country email gender phone special_request roomate_request will_play_in_us_open].sort
```

Comment:
This removes roommate request data from the attendee registration sheets (reports).  If this was unintentional, we could modify the array before merging.

Reply:
Good catch, that was supposed to be an insert not a replacement. I'll add roommate_request back.